tests/foreman/cli/test_remoteexecution.py

```diff
@@ -749,6 +749,78 @@ def test_positive_run_concurrent_jobs(self, registered_hosts, target_sat):
         target_sat.cli.GlobalParameter().delete({'name': param_name})
         assert len(target_sat.cli.GlobalParameter().list({'search': param_name})) == 0
 
+    @pytest.mark.tier3
+    @pytest.mark.no_containers
+    def test_positive_run_serial(self, registered_hosts, target_sat):
+        """Tests subtasks in a job run one by one when concurrency level set to 1
+
+        :id: 5ce39447-82d0-42df-81be-16ed3d67a2a4
+
+        :Setup:
+            0. Create 2 hosts
+
+        :Steps:
+
+            0. Run a bash command job with concurrency level 1
+
+        :expectedresults: First subtask should run immediately, second one after the first one finishes
+
+        :CaseAutomation: Automated
+
+        :CaseLevel: System
+
+        :parametrized: yes
+        """
+        hosts = registered_hosts
+        output_msgs = []
+        template_file = f"/root/{gen_string('alpha')}.template"
+        target_sat.execute(
+            f"echo 'rm /root/test-<%= @host %>; echo $(date +%s) >> /root/test-<%= @host %>; sleep 120; echo $(date +%s) >> /root/test-<%= @host %>' > {template_file}"
```

Comment:
what does the `<%= @host %>` part do? Is this specific to Job Templates?

Reply:
It's an ERB syntax and it outputs hostname